index.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Secreate Scientist</title>
+    <link rel="stylesheet" href="./css/styles.css" />
+  </head>
+  <body>
+    <section class="flex-container">
+      <input class="" type="text" id="textData" />
+
+      <div id="loading-icon"></div>
+
```

Comment:
Unable to use the UI if I put a big string to encrypt "ABCMNOWXYZ"

Please do the following:

- [ ] Give user a way to cancel an ongoing operation
- [ ] Use Web Workers to offload the task away from freezing the main UI thread